### PR TITLE
Innovation week: Updated tutorial experience

### PR DESCRIPTION
**Pull request by jerelmiller**
## Description

Upgrade the guide experience by providing a few new components and supercharging the code blocks. Updates the step components to better align code-blocks with the associated content. 

## Screenshots

Screen Shot 2020-10-23 at 10 08 56 AM
Screen Shot 2020-10-23 at 10 09 02 AM

### CODE REVIEW

File: src/components/MDXContainer.js [hunk 1]
```diff
@@ -4,21 +4,29 @@ import cx from 'classnames';
...
-  Step,
-  Steps,
+  // Remove these when all step/steps components have been updated to the new
+  // name
+  Step: TutorialStep,
+  Steps: TutorialSection,
+  Project,
+  Tutorial,
+  TutorialStep,
+  TutorialSection,
```
Comment (jerelmiller) [inline src/components/MDXContainer.js:29]: I'm not a huge fan of the new names. Would love to get some help on naming these components. We've received feedback that `Steps` and `Step` are too similar and hard to follow. The old names are aliased above, but I'd love to figure out a better name for this component.

**Review by caylahamann — APPROVED** [review]
this looks great to me!!! I'm super excited and I learned a lot reading through :)
